### PR TITLE
Introduce pgbouncer flag for PostgreSQL connector

### PR DESCRIPTION
**Pull request by pimeys**
- Removes `--always-force-transactions`
- Adds a new parameter `pgbouncer=true` to force transactions and `DEALLOCATE ALL` when a transaction starts.

Quaint part: https://github.com/prisma/quaint/pull/133

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 # Prisma Engines
 
-This repository contains a collection of engines that power the core stack for
-[Prisma](https://github.com/prisma/prisma), most prominently [Prisma
-Client](https://github.com/prisma/prisma-client-js) and [Prisma
```
Comment (janpio) [inline README.md]: Looks accidental

File: README.md [hunk 2]
```diff
@@ -76,7 +76,6 @@ USAGE:
     query-engine [FLAGS] [OPTIONS] [SUBCOMMAND]
 
 FLAGS:
-        --always-force-transactions    Runs all queries in a transaction, including all the reads
```
Comment (yoshuawuyts) [inline README.md:79]: cool!

File: query-engine/query-engine/src/exec_loader.rs [hunk 3]
```diff
@@ -63,6 +61,11 @@ async fn postgres(
 
     let psql = PostgreSql::from_source(source).await?;
 
+    let force_transactions = params
```
Comment (janpio) [inline query-engine/query-engine/src/exec_loader.rs:64]: Different name here maybe that is not mixed with the old functionality?
(Assuming this also triggers the additional query)

File: README.md (hunk 1 above)
Comment (pimeys) [reply README.md]: Refresh, was fixed already.

File: query-engine/query-engine/src/exec_loader.rs (hunk 3 above)
Comment (pimeys) [reply query-engine/query-engine/src/exec_loader.rs:64]: Yeah, we still need to pass a boolean flag to the core to force txs. This variable is named to inform why we load the setting from the connection string.

**Pull request merged**